contrib/core-contract-tests/tests/pox-4/pox_StackAggregationCommitSigCommand.ts

```diff
@@ -0,0 +1,148 @@
+import {
+  logCommand,
+  PoxCommand,
+  Real,
+  Stub,
+  Wallet,
+} from "./pox_CommandModel.ts";
+import { Pox4SignatureTopic, poxAddressToTuple } from "@stacks/stacking";
+import { expect } from "vitest";
+import { Cl } from "@stacks/transactions";
+import { bufferFromHex } from "@stacks/transactions/dist/cl";
+
+/**
+ * The `StackAggregationCommitSigCommand` allows an operator commits partially
+ * stacked STX and allocate a new PoX reward address slot. This allows a
+ * stacker to lock fewer STX than the minimal threshold in multiple transactions,
+ * so long as:
+ *  1. The pox-addr is the same.
+ *  2. This "commit" transaction is called _before_ the PoX anchor block.
+ *
+ * This command calls `stack-aggregation-commit` using an `signature`.
+ *
+ * Constraints for running this command include:
+ * - The Operator must have locked STX on behalf of at least one stacker.
+ * - The total amount previously locked by the Operator on behalf of the
+ *   stackers has to be greater than the uSTX threshold.
+ * - All of the Stackers must have delegated to the same pox address.
```

Comment:
Same comment as Friedger above re: "All of the stackers must have..." as a requirement.